include/openenclave/internal/datetime.h

```diff
@@ -58,6 +58,16 @@ oe_result_t oe_datetime_now(oe_datetime_t* value);
  */
 void oe_datetime_log(const char* msg, const oe_datetime_t* date);
 
+/**
+ * Convert oe datetime to time_t.
```

Comment:
```suggestion
 * Convert oe datetime to time_t.
 * NOTE: This function is insecure, since it relies on untrusted time information from the host.
```

Reply:
Same as above, the time is secure or not depends on enclave-side or host-side, in enclave-side verification, it's a secure time actually.